client/src/components/Skills/FilterStyledComponent.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components"
+
+const DivStyled = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  margin: auto;
```

Comment:
collapse the margins into one line.
```suggestion
  margin: 5px auto 10px;
```